conf/code-lists/errors-dms-rej-list.csv

```diff
@@ -0,0 +1,142 @@
+CDS10001,Obligation error: Mandatory Data Element has not been provided.
+CDS10002,Obligation error: Data Element is not allowed.
+CDS10003,Obligation error: Number of Occurrences exceeds what is allowed for the Data Element.
+CDS10004,Commodity code error: Mandatory data element not provided or format error or number of occurrences exceeds what is allowed for the data element.
+CDS10010,Format error: Data element has invalid format.
+CDS10020,Domain error: Data element contains invalid value.
+CDS10021,Domain error: value can not be lower than 0.
+CDS10044,Document is not allowed.
+CDS10045,Calculated customs value is not allowed to be below zero.
+CDS11003,Relation error: The value of ItemTotal is invalid; it does not correspond to the number of times that ItemNumber is used in the declaration.
+CDS11004,Relation error: The combination of the requested procedure followed by a previous procedure followed by an additional procedure code is invalid.
+CDS11005,Relation error: The requested procedure is not the same for each item in the declaration.
+CDS11006,Relation error: The NetMass per item or the sum of all in the declaration is larger than the gross mass.
+CDS11007,Relation error: The submitter in the request/supplementary declaration differs from the related declaration.
+CDS11012,Relation error: element is not allowed to be amended through this additional message type.
+CDS12003,Uniqueness error: Reuse of submitter reference is not (yet) allowed.
+CDS12005,Authorization error: Party ID unknown or invalid.
+CDS12006,Authorization warning: invalid import/export license number.
+CDS12007,Authorization error: authorization not valid for declaration.
+CDS12008,Authorization error: requested amount or quantity is not available for authorization reservation.
+CDS12014,"Sequence error: Sequence does not comply with one of the following conditions:
+- A Goods Presentation Notification is only permitted after an advance declaration or an advance incomplete declaration.
```

Comment:
How does the CSV Reader handle new lines like this?

Reply:
In the same way like you can see here, so when you retrieve this you will receive the same format and everything in string

Reply:
As long as it works :D

Reply:
Added test for this case